packages/node-cli/src/utils/FolderUtils.js

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Since FolderUtils contain code mainly for File Cabinet related process, I was thinking if it would be a good idea to move contents of FolderUtils to FileCabinetService. What do you think?

Reply:
Yeah, I thought about that too. However, I have seen that FileCabinetService is more focused in the internal FileCabinet folder, and only used for UploadFile command. I would try to merge them though

Reply:
Finally I have renamed it AccountFileCabinetService, to differentiate from FileCabinetService... because FleCabinetService make me create a constructor with the local path, when this is not needed when listing files from the server.

I´m not very happy with the name though so I could rename it differently if you think in a better option.